test/e2e/framework/framework.go

```diff
@@ -348,7 +347,7 @@ func (f *Framework) AfterEach() {
 					Logf(summaries[i].PrintHumanReadable())
 				} else {
 					// TODO: learn to extract test name and append it to the kind instead of timestamp.
-					filePath := path.Join(TestContext.ReportDir, summaries[i].SummaryKind()+now.Format(time.RFC3339)+".txt")
+					filePath := path.Join(TestContext.ReportDir, summaries[i].SummaryKind()+"_"+f.BaseName+".txt")
```

Comment:
You can't replace it only with `f.BaseName`, as we may end up running few version of the same test, i.e. have few runs sharing `BaseName`. There's a need for something to distinguish those runs (e.g. timestamp).